config/manager/kustomization.yaml

```diff
@@ -12,5 +12,4 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 images:
 - name: controller
-  newName: controller
-  newTag: latest
+  newName: quay.io/rhgkurz/openshift-sandboxed-containers-operator
```

Comment:
I think this is unrelated to this PR ?

Reply:
> I think this is unrelated to this PR ?

My bad, I forgot to `git reset --hard` _again_ before pushing... I hate `go` for this. :-\

Reply:
Fixed @littlejawa .